fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -1456,6 +1463,47 @@ private void computeScanRangeAssignmentByScheduler(
         }
     }
 
+    private void computeScanRangeAssignmentWithMinNodeStrategy(
```

Comment:
Could you please add some comment to explain the logic of this method?

Reply:
comments resolved